test/v1/configuration.go

```diff
@@ -203,3 +203,19 @@ func CheckConfigurationState(client *test.ServingClients, name string, inState f
 func IsConfigurationReady(c *v1.Configuration) (bool, error) {
 	return c.IsReady(), nil
 }
+
+// GetConfiguration gets a configuration by name
+func GetConfiguration(clients *test.Clients, configurationName string) (configuration *v1.Configuration, err error) {
```

Comment:
This remains unused then? Should we delete it?

Reply:
you are 100% correct.. this is now removed